bn-expansions/src/EM/selective-discards.ts

```diff
@@ -3,34 +3,44 @@ import { flattenDeep, get, set } from 'lodash';
 import { groupingsToSpeciesDummyMap } from './helpers/maps';
 import { Catches, Disposition, sourceType } from '@boatnet/bn-models';
 
-const expansionCatchIds = ['5000']; //TODO maybe get this from some couch view or keep as a hardcoded value
+const dbConfig = require('../../dbConfig.json').dbConfig;
```

Comment:
Curious how this interacts with the TripsApi dbConfig (when deployed) - is this the same location as for TripsApi / same file?  Or would this mean bn-expansions needs it's own config now.

If not the same file, It may make more sense for bn-expansions to call a function in tripsApi to query couchDB.

Reply:
so originally I had a dbConfig file here in bn-expansions. It's probably not a good idea to have two configs since they can get out of sync. So I moved the database queries to TripsAPI and updated expansion interface to take a mixedGroupings argument. Here's my code changes in the TripsAPI https://github.com/nwfsc-fram/TripsApi/pull/87/files